mindmeld/components/question_answerer.py

```diff
@@ -304,18 +367,41 @@ def transform(doc):
                     app_namespace,
                 )
 
+        def _generate_mapping_data(embedder_model, embedding_fields):
+            # generates a dictionary with any metadata needed to create the mapping"
+            MAX_ES_VECTOR_LEN = 2048
+            embedding_properties = []
+            mapping_data = {"embedding_properties": embedding_properties}
+
+            dims = len(embedder_model.get_encodings(["encoding"])[0])
+            if dims > MAX_ES_VECTOR_LEN:
```

Comment:
is there a limit on the size of the vector itself (in terms of memory)?

Reply:
Not sure I understand - how are memory and length limits different here?